src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -102,6 +102,12 @@ class NativeSessionStorage implements SessionStorageInterface
      */
     public function __construct(array $options = array(), $handler = null, MetadataBag $metaBag = null)
     {
+        $this->setMetadataBag($metaBag);
+
+        if (PHP_VERSION_ID >= 70200 && \PHP_SESSION_ACTIVE === session_status()) {
```

Comment:
I'd suggest to remove the `PHP_VERSION_ID >= 70200` part: the behavior should be consistent across versions.